frontend/src/features/permits/components/permit-list/BasePermitList.tsx

```diff
@@ -139,6 +139,7 @@ export const BasePermitList = ({
       // Search Bar
       positionGlobalFilter="left"
       initialState={{ showGlobalFilter: true }} //show the search bar by default
+      globalFilterFn="contains" // use contains vs the default fuzzy filter fn
```

Comment:
the default `fuzzy` filter function is incredibly fuzzy. It will return rows that don't seem to always match the filter keywords